agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -1411,6 +1411,29 @@ bool slave_thread::handle_cmdu_backhaul_manager_message(
 
         break;
     }
+    case beerocks_message::ACTION_BACKHAUL_HOSTAP_STATS_MEASUREMENT_REQUEST: {
+        LOG(DEBUG) << "ACTION_BACKHAUL_HOSTAP_STATS_MEASUREMENT_REQUEST";
+        if (monitor_socket) {
+            auto request_in = beerocks_header->addClass<
+                beerocks_message::cACTION_BACKHAUL_HOSTAP_STATS_MEASUREMENT_REQUEST>();
+            if (request_in == nullptr) {
+                LOG(ERROR) << "addClass cACTION_BACKHAUL_HOSTAP_STATS_MEASUREMENT_REQUEST failed";
+                return false;
+            }
+
+            auto request_out = message_com::create_vs_message<
+                beerocks_message::cACTION_MONITOR_HOSTAP_STATS_MEASUREMENT_REQUEST>(
+                cmdu_tx, beerocks_header->id());
+            if (request_out == nullptr) {
+                LOG(ERROR) << "Failed building message!";
+                return false;
+            }
+            request_out->sync() = request_in->sync();
+            LOG(DEBUG) << "Will now send ACTION_MONITOR_HOSTAP_STATS_MEASUREMENT_REQUEST";
```

Comment:
the "will now" is redundant.
`LOG(DEBUG) << "send ACTION_MONITOR_HOSTAP_STATS_MEASUREMENT_REQUEST";`